impl/proto/monitorserver_v1_service/monitorserver_v1_service.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// Monitor Server Service
+//
+// The Key Transparency monitor server service consists of APIs to fetch
+// mutations.
+package monitorserver.v1.service;
+
+import "github.com/google/keytransparency/core/proto/keytransparency_v1_types/keytransparency_v1_types.proto";
+import "google/api/annotations.proto";
+
+// The MonitorService API represents a directory of mutations:
+//
+// - Mutations can be collected using the GetMutation APIs. Mutation resources
+//   are named /v1/mutations/* and /v1/mutations/*/stream.
+//
+service MonitorService {
+  // GetMutations returns a list of mutations in a specific epoch.
+  //
+  // Returns a list of mutations and their inclusion proofs along with the epoch
+  // signed map root.
+  rpc GetMutations(keytransparency.v1.types.GetMutationsRequest)
+      returns (keytransparency.v1.types.GetMutationsResponse) {
+    option (google.api.http) = { get: "/v1/epochs" };
+  }
+
+  // GetMutationsStream is a streaming API similar to GetMutations.
+  //
+  // Returns a list of mutations and their inclusion proofs along with the epoch
+  // signed map root.
+  rpc GetMutationsStream(keytransparency.v1.types.GetMutationsRequest)
```

Comment:
If I recall correctly, the reason we need a separate proto is only because of this streaming RPC.

Would it make sense to split the protos up based on the streaming rpcs? 
eg, keep GetMutations along with the normal API and create a proto and service just for streaming RPCs? eg `service StreamingRPCs`